it/connectors/CreateOrAmendSavingsTysConnectorISpec.scala

```diff
@@ -39,8 +39,7 @@ class CreateOrAmendSavingsTysConnectorISpec extends PlaySpec with WiremockSpec {
   lazy val connector: CreateOrAmendSavingsTysConnector = app.injector.instanceOf[CreateOrAmendSavingsTysConnector]
   lazy val httpClient: HttpClient = app.injector.instanceOf[HttpClient]
 
-  val nino = "nino"
```

Comment:
I think this may have been better to be kept as a variable as it is used to many times below.  If you agree can you change it please.

Reply:
Yep no problem, have changed in the next commit.